collector/metrics.go

```diff
@@ -74,7 +80,22 @@ func (e *Exporter) Collect(c chan<- prometheus.Metric) {
 		}
 	} else {
 		e.up.With(prometheus.Labels{"host": e.hostname}).Set(0)
-		log.Println("Could not fetch metrics from squid instance: ", err)
+		log.Println("Could not fetch counter metrics from squid instance: ", err)
 	}
+
+	insts, err = e.client.GetServiceTimes()
+
+	if err == nil {
+		e.up.With(prometheus.Labels{"host": e.hostname}).Set(1)
```

Comment:
I think we don't need to call up twice, it's probably fine if you remove this one, in case we encounter error while fetching service times, if the counters already succeeded, the issue is probably not related to squid server being down. The error message at the end is probably suffice.

Reply:
This make sense. Updated 👍